lib/DDG/Meta/Data.pm

```diff
@@ -162,9 +174,13 @@ sub get_ia {
 }
 
 sub get_js {
-    my ($self, $id) = @_;
+    my ($self, $by, $lookup) = @_;
+    my $ia = $self->get_ia($by => $lookup);
+    return unless $ia;
 
-    my $metaj = eval { JSON::XS->new->ascii->encode($self->get_ia(id => $id)) } || qq|{"encode_json error":"$@"}|;
+    my $metaj;
+    my $id = $ia->{id};
+    eval { $metaj = JSON::XS->new->ascii->encode($ia) } || return;
```

Comment:
It was before
``` Perl
my $metaj = eval { ....
```
not sure why that changed.  Also, if we're going to allow $by to be 'module' now, you need to check for $ia being an array.   Do we really need to allow this?